File: Homeworks/homework009.py
#dictionary
Person = {'personID': 0,
          'firstName': "",
          'lastName': "",
          'Account': {'accountNumber': 0,
                      'accountType': 0,
                      'money': 0,
                      'limit': 0}
          }

def inputPerson():
    Person['personID'] = int(input("Enter Customer ID: "))
    Person['firstName'] = str(input("Enter First Name: "))
    Person['lastName'] = str(input("Enter Last Name: "))
    
    #account detail
    Person['Account']['accountNumber'] = int(input("Enter Account number: "))
    Person['Account']['accountType'] = int(input("Enter Account type (1) or (2): "))
    accountType = Person['Account']['money'] = int(input("Enter initial money: "))
    #check account type 2
    if(Person['Account']['accountType'] == 2):
        Person['Account']['limit'] = int(input("Enter limit: "))
   
    
def showPerson():
    print("Customer id: %d" %(Person['personID']))
    print("Customer First Name: %s" %(Person['firstName']))
    print("Customer Last Name: %s" %(Person['lastName']))
    print("Account number: %d" %(Person['Account']['accountNumber']))
    if(Person['Account']['accountType'] == 1):
        print("Account type: (1) Saving account")
    else:
        print("Account type: (2) Current account")
    print("Balance: %d" %(Person['Account']['money']))
    if(Person['Account']['accountType'] == 2):
        print("Limit: %d" %(Person['Account']['limit']))

def deposite():
    money = int(input("Please enter the amount you want to deposit: "))
    Person['Account']['money'] = Person['Account']['money'] + money;

def withdraw():
    money = int(input("Please enter the amount you wish to withdraw: "))
    Person['Account']['money'] = Person['Account']['money'] - money;
    
def main():
    inputPerson()
    
    while(True):
        print("----------------------------------------------------")
        print("    (1) Deposite   (2) Withdraw   (3) Exit     ")
        print("----------------------------------------------------")
        
        choice = int(input())
        
        if(choice == 1):
            deposite()
        elif(choice == 2):
            withdraw()
        elif(choice == 3):
            showPerson()
            break
        else:
            break

main()

File: Homeworks/homework009.cpp
#include <bits/stdc++.h>
using namespace std;

typedef long int ll;

void input();
void deposit();
void withdraw();
void show();

struct Account{
	long long int accountID;
	int accountType;
	ll money = 0;
	ll limit = 0;
};

struct Person{
	int personID;
	char firstName[100];
	char lastName[100];
	struct Account account;
} customer_account;

int main(){
	int choice;
	
	input();
	
	while(true){
		cout << "------------------------------------------------------\n";
		cout << "(1) Deposit   " << "(2) Withdraw   " << "(3) Exit" << endl;
		cout << "------------------------------------------------------\n";
		
		cin >> choice;
		
		if(choice == 1) deposit();
		else if(choice == 2) withdraw();
		else if(choice == 3){
			show();
			break;
		}else break;
	}
}

void input(){
	int type(0);
	ll limit(0);
	
	cout << "Customer id: "; cin >> customer_account.personID;
	cout << "Enter first name: "; cin >> customer_account.firstName;
	cout << "Enter last name: "; cin >> customer_account.lastName;
	//account details
	cout << "Enter account number: "; cin >> customer_account.account.accountID;
	cout << "Choose account type (1) or (2): "; cin >> type;
	cout << "Enter initial money: "; cin >> customer_account.account.money;
	
	//limit
	if(type == 2){
		cout << "Enter Limit: "; cin >> limit;
	}	
}

void show(){
	cout << "Customer id: " << customer_account.personID << endl;
	cout << "Customer First Name: " << customer_account.firstName << endl;
	cout << "Customer Last Name: " << customer_account.lastName << endl;
	cout << "Account number: " << customer_account.account.accountID << endlCus;
	if(customer_account.account.accountType == 1){
		cout << "Account type: (1) Savings account" << endl;
	}else{
		cout << "Account type: (2) Current account" << endl;
	}
	cout << "Balance: " << customer_account.account.money << endl;
	if(customer_account.account.accountType == 2){
		cout << "Limit: " << customer_account.account.limit << endl;
	}
	
}

void deposit(){
	ll money;
	cout << "Please enter the amount you want to deposit: "; cin >> money;
	customer_account.account.money += money;
}

void withdraw(){
	ll money;
	cout << "Please enter the amount you wish to withdraw: "; cin >> money;
	customer_account.account.money -= money; 
}
